rplugin/python3/deoplete/__init__.py

```diff
@@ -1,4 +1,4 @@
-#=============================================================================
+# ============================================================================
```

Comment:
Why did you added the changes?
If it is needed for the coding style, please add in the all python files.